nimbus-features/libraryCoordinatorRefactor.yaml

```diff
@@ -15,5 +15,5 @@ features:
           enabled: false
       - channel: developer
         value:
-          enabled: false
```

Comment:
We should turn ON in Beta and by default too 🙏 same for the other PR

Reply:
Ok i'm doining it now